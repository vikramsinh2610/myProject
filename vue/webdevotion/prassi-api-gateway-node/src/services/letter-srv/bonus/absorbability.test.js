const Absorbability = require('./absorbability');

describe('Absorbability', () => {
  test('should have these properties', () => {
    const absorbability = new Absorbability({
      indirectProductionPercentage: 3000,
      directProductionPercentage: 7000,
    });
    expect(absorbability).toHaveProperty('indirectProductionPercentage');
    expect(absorbability).toHaveProperty('directProductionPercentage');
  });

  test('should return it s JSON Schema', () => {
    expect(Absorbability.getJSONSchema()).toMatchSnapshot();
  });
});
