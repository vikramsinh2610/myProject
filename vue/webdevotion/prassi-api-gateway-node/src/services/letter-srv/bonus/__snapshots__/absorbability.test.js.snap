// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Absorbability should return it s JSON Schema 1`] = `
Object {
  "properties": Object {
    "directProductionPercentage": Object {
      "description": "Direct production percentage, with 2 decimals as integer",
      "maximum": 10000,
      "minimum": 0,
      "type": "integer",
    },
    "indirectProductionPercentage": Object {
      "description": "Indirect production percentage, with 2 decimals as integer",
      "maximum": 10000,
      "minimum": 0,
      "type": "integer",
    },
  },
  "required": Array [
    "directProductionPercentage",
    "indirectProductionPercentage",
  ],
  "type": "object",
}
`;
